src/components/Toggle/Toggle.scss

```diff
@@ -180,11 +205,20 @@
 .ms-Toggle-field {
   position: relative;
   display: inline-block;
-  width: 57px;
+  width: 45px;
   height: 20px;
   box-sizing: border-box;
-  border: 2.5px $ms-color-neutralTertiaryAlt solid;
+  border: 2px solid $ms-color-neutralTertiary;
+  border-radius: 20px;
   cursor: pointer;
+  transition-property: background, left;
+  transition-duration: 250ms;
+  transition-timing-function: cubic-bezier(.4, 0, .23, 1);
```

Comment:
Same here--I won't block approval on this, but I'm wondering if we should just use one of `$ms-ease1` or `$ms-ease2`.

Otherwise, we should probably either:
1. Capture this as a local variable for Toggle.scss (probably the safest for now), or
2. Incorporate this as a global animation